zib/StructureDefinitions/zib-AbilityToDrink.DrinkingLimitations.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-AbilityToDrink.DrinkingLimitations"/>
+  <text>
+    <status value="empty" /><div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div></text>
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-AbilityToDrink.DrinkingLimitations"/>
+  <name value="ZibAbilityToDrinkDrinkingLimitations"/>
+  <title value="zib AbilityToDrink.DrinkingLimitations"/>
+  <status value="draft"/>
+  <publisher value="Nictiz"/>
+  <contact>
+    <name value="Nictiz"/>
+    <telecom>
+      <system value="url"/>
+      <value value="https://www.nictiz.nl"/>
+      <use value="work"/>
+    </telecom>
+  </contact>
+  <description value="DrinkingConstraints specifies the patient’s constraints in drinking."/>
```

Comment:
The zib concept decription is : "DrinkingLimitations specifies the patient’s limitations in drinking."

Reply:
Fixed!